sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/LiveMetricConstants.cs

```diff
@@ -27,9 +27,13 @@ internal static class MetricId
             // EXCEPTIONS
             internal const string ExceptionsPerSecondMetricIdValue = @"\ApplicationInsights\Exceptions/Sec";
 
-            // PROCESS METRICS
+            // PROCESS METRICS (OLD) // TODO: Remove these after the UX is updated to use the new metrics
             internal const string MemoryCommittedBytesMetricIdValue = @"\Memory\Committed Bytes";
             internal const string ProcessorTimeMetricIdValue = @"\Processor(_Total)\% Processor Time";
+
+            // PROCESS METRICS (NEW)
+            internal const string ProcessPhysicalBytesMetricIdValue = @"\Process\Physical Bytes";
```

Comment:
If UX is not ready, why do we need to send additional metrics now? Shouldn't this be added once UX is ready to respect this metrics.

Reply:
This was seen as the most reasonable compromise. 
We considered three options. 
I favored Option A and the service team agreed.

- Option A:
SDKs send Process Metrics using both old and new names.
SDKs will drop old names after UX changes are in Production.
Pros: no interruption to cx experience. Facilitates necessary testing for SDKs.
Cons: will require SDKs to do some minor cleanup at a later date

- Option B:
SDKs send Process Metrics using old names while in beta.
switch to new names for stable release
Pros: facilitates necessary testing for SDKs.
Cons: customers may have no data until UX makes changes

- Option C:
SDKs send Process Metrics using only new names.
Cons: SDKs unable to test UX. Cx will have no-data until UX makes changes.

---
Addressing your concern, we did not consider only sending the old metric until the UX change is in place.
But the way I see it:
Pros: SDK not sending extra data.
Cons: UX doesn't have valid data to test new name. SDK must make a change at a later date.

Reply:
Don't think we need to worry about sending couple of extra metrics in LiveMetrics. Its only sent on-demand, and also has no billing implications...

Reply:
I was worried about the payload size as it is sent every 1 seconds. As we are in beta, we could try adding these metrics.